app/javascript/src/components/Authentication/Signup.jsx

```diff
@@ -43,67 +32,64 @@ const Signup = ({ history }) => {
           Signup
         </h2>
         <Formik
-          initialValues={formInitialValues.signupForm}
+          initialValues={SIGNUP_FORM_INITIAL_VALUES}
           validateOnBlur={submitted}
           validateOnChange={submitted}
-          onSubmit={onSubmit}
-          validationSchema={formValidationSchemas.signupForm}
+          onSubmit={handleSubmit}
+          validationSchema={SIGNUP_FORM_VALIDATION_SCHEMA}
         >
-          {({ handleSubmit }) => (
-            <Form className="w-full space-y-6 rounded-md border bg-white p-8 shadow">
+          {({ isSubmitting }) => (
+            <Form className="w-full p-8 space-y-6 bg-white border rounded-md shadow">
               <Input
+                required
                 name="email"
                 type="email"
                 label="Email"
                 placeholder="oliver@example.com"
-                required
               />
               <Input
+                required
                 name="firstName"
                 type="text"
                 label="First name"
-                placeholder="Sam"
-                required
+                placeholder="Oliver"
               />
               <Input
+                required
                 name="lastName"
                 type="text"
                 placeholder="Smith"
                 label="Last name"
-                required
               />
               <Input
+                required
                 name="password"
                 type="password"
                 label="Password"
                 placeholder="******"
-                required
               />
               <Input
+                required
```

Comment:
Do we need this `required` statement over here if the Yup validation already requires it? Can you check on how it's different?